exts/cesium.omniverse/cesium/omniverse/extension.py

```diff
@@ -217,6 +218,11 @@ def _on_update_frame(self, _):
             viewport.height = float(viewport_api.resolution[1])
             viewports.append(viewport)
 
+        if len(viewports) != self._num_credits_viewport_frames:
+            self._logger.info("CreditViewportFrame: num viewports changed, triggering CreditsViewportFrames setup")
```

Comment:
I know we're pretty inconsistent about this, but it would be better to log this as a debugging message rather than as an informational one, since it's in the frame update.

Reply:
True. That would be better. What if we just ditch it? I don't see it being of any use to an end user.

Reply:
I'm fine with ditching it entirely.